.github/workflows/smoke-checks.yml

```diff
@@ -223,7 +223,7 @@ jobs:
     env:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       GITHUB_PR_NUM: ${{ github.event.number }}
-      XCODE_VERSION: "14.0.1"
+      XCODE_VERSION: "15.0.1"
```

Comment:
15.4?

Reply:
less supported is 15.0.1 and this is the one we use in `build-xcode15` job